apis/management.cattle.io/v3/logging_types.go

```diff
@@ -38,9 +38,13 @@ type ProjectLogging struct {
 }
 
 type LoggingCommonField struct {
-	DisplayName         string            `json:"displayName,omitempty"`
-	OutputFlushInterval int               `json:"outputFlushInterval,omitempty" norman:"default=60"`
-	OutputTags          map[string]string `json:"outputTags,omitempty"`
+	DisplayName           string            `json:"displayName,omitempty"`
+	OutputFlushInterval   int               `json:"outputFlushInterval,omitempty" norman:"default=60"`
+	OutputTags            map[string]string `json:"outputTags,omitempty"`
+	EnableMultiLineFilter bool              `json:"enableMultiLineFilter,omitempty"`
+	MultiLineStartRegexp  string            `json:"multiLineStartRegexp,omitempty"`
+	ContinueLineRegexp    string            `json:"continueLineRegexp,omitempty"`
```

Comment:
Continues is adjective, use continues as [plugin](https://github.com/fluent-plugins-nursery/fluent-plugin-concat) did?